hubble-relay/cmd/root.go

```diff
@@ -22,8 +22,8 @@ const configFilePath = "/etc/hubble-relay/config.yaml"
 func New() *cobra.Command {
 	rootCmd := &cobra.Command{
 		Use:          "hubble-relay",
-		Short:        "hubble-relay is a proxy server for the hubble API",
-		Long:         "hubble-relay is a proxy server for the hubble API.",
+		Short:        "Hubble-relay is a proxy server for the hubble API",
+		Long:         "Hubble-relay is a proxy server for the hubble API.",
```

Comment:
If we're referring to the command name (i.e. the `Use:` part of the `cobra.Command`) then it should stay all lower case but here I think we should rather
```suggestion
		Short:        "Hubble Relay is a proxy server for the hubble API",
		Long:         "Hubble Relay is a proxy server for the hubble API.",
```

Reply:
@kaworu 
thanks very much for review.
Updated.